src/components/events/EventCreateUpdate.vue

```diff
@@ -218,26 +223,24 @@ export default {
     location: { required },
     account: {},
   },
-  mounted() {
+  async mounted() {
```

Comment:
Theoretically, this lifecycle is synchronous. According to Evan You ", this requires a fundamental rethink/rewrite of the architecture to achieve, and can potentially break a lot of logic that relies on the synchronous nature of lifecycle hooks". https://github.com/vuejs/vue/issues/7209

Reply:
Ok, so, move the async code to a different async function to improve the readability.

Reply:
Great!!